api/proto/core/pipeline/definition/definition.proto

```diff
@@ -52,6 +52,21 @@ service DefinitionService {
       get: "/api/pipeline-definitions/actions/statistics-group-by-filePath",
     };
   }
+  rpc UpdateExtra (PipelineDefinitionExtraUpdateRequest) returns (PipelineDefinitionExtraUpdateResponse) {
+    option (google.api.http) = {
+      put: "/api/pipeline-definition-extra/{pipelineDefinitionExtraID}",
```

Comment:
/api/pipeline-definitions/{pipelineDefinitionID}/actions/update-extra
I think no need to expose extra id to invoker.